sympy/physics/mechanics/functions.py

```diff
@@ -11,6 +11,7 @@
 from sympy import simplify
 from sympy.core.backend import (Matrix, sympify, Mul, Derivative, sin, cos,
                                 tan, AppliedUndef, S)
+from sympy.matrices.immutable import ImmutableDenseMatrix
```

Comment:
Is this import used anywhere?

Reply:
No, its not being used anywhere. I have removed it.